test-suite/src/test/scala/org/scalajs/testsuite/javalib/LinkedListTest.scala

```diff
@@ -38,7 +38,7 @@ abstract class LinkedListTest[F <: LinkedListFactory](listFactory: F) extends Ab
       ll.addFirst(1)
       ll.removeLast()
       ll.addFirst(2)
-      expect(ll.peekLast()).toEqual(2)
+      expect(ll.peekLast()).toEqual(1)
```

Comment:
Hum, no, don't change this. `clear()` the list in the middle, or better yet, create a new one.
Otherwise this test does not test the same thing as before.